fxgl/src/main/kotlin/com/almasb/fxgl/dsl/FXGL.kt

```diff
@@ -589,6 +606,39 @@ class FXGL private constructor() { companion object {
                 .also { addUINode(it, x, y) }
     }
 
+    /**
+     * Returns new Text initialized with property value, whose name was passed to the function
+     * @param propertyName name of the property in PropertyMap
+     * @param x x coordinate
+     * @param y y coordinate
+     * @throws IllegalArgumentException if property was not found in PropertyMap
+     * @return Text()
+     */
+    @JvmStatic fun addPropText(propertyName: String, x: Double, y: Double) : Text {
```

Comment:
This should be renamed to `@JvmStatic fun addVarText(varName: String, x: Double, y: Double): Text`. There is already a function with this signature. That function (which only worked for integer properties) should be replaced with this new one.

Reply:
ok, done